schematypes_test.go

```diff
@@ -90,11 +102,11 @@ func TestLaunchAttachRequestInterface(t *testing.T) {
 			},
 			Command: "attach",
 		},
-		Arguments: map[string]interface{}{"foo": "bar"},
+		Arguments: toRawMessage(map[string]interface{}{"foo": "bar"}),
 	}
 
 	f := func(r LaunchAttachRequest) (int, string, interface{}) {
-		return r.GetSeq(), r.GetRequest().Command, r.GetArguments()["foo"]
+		return r.GetSeq(), r.GetRequest().Command, fromRawMessage(r.GetArguments())["foo"]
```

Comment:
If we are not mapping fields to a map anymore, but leave the raw bytes as-is, shouldn't you be changing the test to test that instead?